R/data-importer-configuration.R

```diff
@@ -6,7 +6,7 @@
 DataImporterConfiguration <- R6::R6Class(
   "DataImporterConfiguration",
   inherit = DotNetWrapper,
-  cloneable = TRUE,
+  cloneable = FALSE,
```

Comment:
Why are we doing this explicitley since the base DotNetWrapper is FALSE? I am confused

Reply:
> I am confused

So was I. But if we not specify `cloneable` explicitely, NOT the property of the parent class but `TRUE` is used by default. See here: 

https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/pull/82#discussion_r819474906

Reply:
ok I See. Thanks @PavelBal